digital_comms/fixed_network/model.py

```diff
@@ -697,6 +697,7 @@ def upgrade(self, action):
     def update_desirability_to_adopt(self, desirability_to_adopt):
         self.adoption_desirability = desirability_to_adopt
 
+
```

Comment:
Assuming this is a PEP8 style preference to have a double line space after a class or function?